tests/test_helpers.py

```diff
@@ -74,3 +74,70 @@ def f(s):
         with warnings.catch_warnings():
             warnings.simplefilter("ignore")
             self.assertTrue(f(s).index.equals(s_true.index))
+
+    """
+    InputSeries.
+    """
+
+    def test_inputseries_function_executes_correctly(self):
+        @_helper.InputSeries(_helper.TextSeries)
+        def f(s, t):
+            return t
+
+        s = pd.Series("I'm a TextSeries")
+        t = "test"
+        self.assertEqual(f(s, t), t)
+
+    def test_inputseries_wrong_type(self):
+        @_helper.InputSeries(_helper.TextSeries)
+        def f(s):
+            pass
+
+        self.assertRaises(TypeError, f, pd.Series([["token", "ized"]]))
+
+    def test_inputseries_correct_type_textseries(self):
+        @_helper.InputSeries(_helper.TextSeries)
+        def f(s):
+            pass
+
+        try:
```

Comment:
Why do you have the `try`/`except` there? Might be overcomplicated, isn't it?

Reply:
We want to make sure that calling it with the right type does not raise an Exception here. I'm going off of [this stackoverflow discussion](https://stackoverflow.com/questions/4319825/python-unittest-opposite-of-assertraises). We could just do e.g. `f(pd.Series("I'm a TextSeries"))`, but if that raises an Exception, we get an _exception_ in the test suite which makes us think a test is written wrong, instead of correctly getting a _failure_  telling us "test is programmed correctly, but it's failing".

Reply:
Thank you! Great job!